components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -15,13 +15,15 @@ import Link from 'next/link';
  * @param props.linkUrl - the url to the user's entry page
  * @param props.isEliminated - If true, the user is flagged as eliminat4ed
  * @param props.isPickSet - if true, the team logo of the picked team shows up on the LeagueEntries card and the button changes from "make a pick" to "chagne pick"
+ * @param props.teamLogo - the url to the team logo
  * @returns {React.JSX.Element} - A div element that contains the user's entry information
  */
 const LeagueEntries = ({
   entryName,
   linkUrl,
   isEliminated = false,
   isPickSet = false,
+  teamLogo = '',
```

Comment:
Do not default this to an empty string. Make it required. We will need to update our DB with the remaining NFL team logos.

Reply:
Updated to not default to an empty string.

Reply:
It's still empty. 
```suggestion
  teamLogo,
```

Reply:
@shashilo the reason why I made it empty is because if teamLogo is optional, it will default to an empty string, similar to `isPickSet` and `isEliminated`. Please let me know what your thoughts are. Thanks!